Client/qtTeamTalk/streammediafile.ui

```diff
@@ -42,7 +45,7 @@
            <string>Browse</string>
           </property>
           <property name="text">
-           <string>...</string>
+           <string>... &amp;B</string>

```

Comment:
Should we instead change this to an icon as well (same as Refresh button)? It looks a bit weird with "...B"